package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "urls": [
+    ["rtttl.py", "github:mchobby/upy-rtttl/rtttl.py"]
+    ["songs.py", "github:mchobby/upy-rtttl/songs.py"]
```

Comment:
This should point into this repository, rather than an external repository. I also see several commits updating this file. I think you could also squash all of the commits on this PR into a single commit.